.gitignore

```diff
@@ -5,3 +5,4 @@ coverage
 conf/local.js
 *.log
 .DS_Store
+.vscode
```

Comment:
Should be fine. Anything specific to our environments/IDE's can go here. so we don't accidentally commit stuff we shouldn't.